code/src/components/TaskAdd.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import uniqid from 'uniqid'
+import styled from 'styled-components'
+import tasks from 'reducers/tasks'
+
+const AddForm = styled.form`
+display: flex;
+position: relative;
+background-color: white;
+margin-bottom: 25px;
+`
+const ButtonAdd = styled.button`
+border: none;
+background: none;
+font-size: 2rem;
+margin-left: 10px;
+cursor: pointer;
+`
+const InputField = styled.input`
+font-size: 1.5em;
+border: none;
+outline-style: none;
+width: 100%;
+height: 5rem;
+cursor: pointer;
+`
```

Comment:
Super nice - great division and order for your imports, and the naming for your styled components is really clear. I like that you put your styled components in the beginning - I put mine in the end, but it's a lot nicer reading them first. I think I should possibly switch up my approach and do what you're doing instead, because I see that it gives better context for what I'm about to read in the render. Super cool reading your code and being able to reflect over how I can improve my own.

I also like that you wrapped the whole thing in a form - makes it more accessible and semantically correct. Also means that I can press the enter button to submit a task, without you needing to write a function for it. I think my stubbornness with my own to-do list app ended up sacrificing a lot of these important features and was forced to re-create them with extra functions, role=, etc.